static-site/components/nav/nav-logo.tsx

```diff
@@ -20,21 +26,23 @@ export default function NavLogo(): React.ReactElement | null {
   ) : null;
 }
 
-const Logo = (): React.ReactElement => (
-  <a href="/" className={styles.logo}>
-    <Image src={logo} alt="Nextstrain (logo)" width="40" height="40" />
-  </a>
-);
-
-const LogoType = (): React.ReactElement => {
-  const rainbowTitle = siteTitle
-    .split("")
-    .map((letter, i) => (
-      <span key={i} style={{ color: `var(--titleColor${i})` }}>{letter}</span>
-    ));
-  const SubTitle = (): React.ReactElement => (
-    <div>Groups Server</div>
+/** React Client Component that renders the Nextstrain logo */
+function Logo(): React.ReactElement {
+  return (
+    <a href="/" className={styles.logo}>
+      <Image src={logo} alt="Nextstrain (logo)" width="40" height="40" />
+    </a>
   );
+}
+
+/** React Client Component that renders the title below the logo */
+function LogoType(): React.ReactElement {
```

Comment:
Not a fault of this PR, but I think this would be a more accurate name given the new description.

```suggestion
/** React Client Component that renders the title below the logo */
function LogoTitle(): React.ReactElement {
```

Reply:
i changed the comment to use `logotype` instead.

Reply:
Hmm, why "type" and not "title"? Seems to be a title...

Reply:
"[logotype](https://www.thefreedictionary.com/logotype)" is what it's called in printing; technically this bit of text is a [wordmark](https://en.wikipedia.org/wiki/Wordmark). It's not really a title.

Reply:
Ohh! My mind was in TypeScript land so "type" seemed off. This makes sense now.